src/database/operations/load/loadUserBundle.ts

```diff
@@ -0,0 +1,43 @@
+import { loadBundle } from './loadBundle';
+import type {
+	JsonResultSet,
+	Bundle,
+	Proposal,
+	AuthContext,
+} from '../../../types';
+
+export const loadUserBundle = async (
+	queryParameters: {
+		offset: number;
+		limit: number;
+		authenticationId?: string;
+	},
+	authContext?: AuthContext,
+): Promise<Bundle<Proposal>> => {
```

Comment:
Is this supposed to be a bundle of Proposal or of User?

Reply:
Great catch!  User!  This is why casting types can lead to bugs 😅